models/src/main/thrift/content/v1.thrift

```diff
@@ -901,6 +903,28 @@ struct ListElementFields {
     2: optional ListType type;
 }
 
+struct TimelineEvent {
+    1: required list<BlockElement> body = [];
+
+    2: optional BlockElement main;
```

Comment:
What's the difference between a main element and the body elements?

Reply:
Similar to the `main` block (used for main media) and `body` blocks (used for the article), we will allow a single main element and then have the rest of the content in body. This will make it easier for the platforms to render the first image differently.